semantic_search/schemas.py

```diff
@@ -20,6 +20,7 @@ class Search(BaseModel):
     query: Document
     documents: List[Document] = []
     top_k: int = Field(10, gt=0, description="top_k must be greater than 0")
+    use_docs: bool = False
```

Comment:
Nitpick but I might name this something like `use_docs_only` but it doesn't matter I guess as we are the only ones using this API.

Reply:
`use_docs_only` is fine with me - I tried a few and wasn't sure which one to live with. I didn't want to stomp on top_k because it got really confusing

Reply:
or how bout `docs_only`

Reply:
Yeah that would be fine, I think `only` is the important part